mobile/lib/screens/dashboard/dashboard_view.dart

```diff
@@ -1,3 +1,4 @@
+// ignore_for_file: use_build_context_synchronously
```

Comment:
You might want to omit these changes, as they are empty changes.
Additionally, I saw 3 other files, that seem to be from different PRs, Please review the files changed in this PR to ensure only the desired files were pushed.